src/cmake/modules/GUDHI_third_party_libraries.cmake

```diff
@@ -134,23 +134,25 @@ add_definitions( -DBOOST_SYSTEM_NO_DEPRECATED )
 
 if (WITH_GUDHI_PYTHON)
   # Find the correct Python interpreter.
-  # Can be set with -DPYTHON_EXECUTABLE=/usr/bin/python3 or -DPython_ADDITIONAL_VERSIONS=3 for instance.
-  find_package( PythonInterp )
-  
+  # Can be set with -DPython_EXECUTABLE=/usr/bin/python3 for instance.
+  # CMP0094 OLD finds system python interpreter instead of conda even when using '-DCMAKE_PREFIX_PATH=$CONDA_PREFIX'
+  cmake_policy(SET CMP0094 NEW)
```

Comment:
Maybe the comment could say:
Default Python_FIND_STRATEGY to LOCATION: Stops lookup as soon as a version satisfying version constraints is founded (as opposed to VERSION: Try to find the most recent version in all specified locations.)

(Supposedly Python_FIND_VIRTUALENV defaults to FIRST since 3.15, so if you run cmake in an active conda environment the conda version of python should be selected anyway, but setting the policy to NEW looks good in any case.)

Reply:
I took your comment proposal on [1b4b4fa](https://github.com/GUDHI/gudhi-devel/pull/1112/commits/1b4b4fac2179b0eeaa134c0cf4a84b74823b963a)